sdk/model/extended_workflow_def.go

```diff
@@ -0,0 +1,37 @@
+/*
+ * Orkes Conductor API Server
+ *
+ * Orkes Conductor API Server
+ *
+ * API version: v2
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+package model
+
+type ExtendedWorkflowDef struct {
```

Comment:
let's find a better name?  Extended Workflow Def does not give much.

Reply:
Started using this internally instead of asking the user to pass it.